<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoPointDeVente</class>
 <widget class="QDialog" name="InfoPointDeVente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Informations sur le point de vente</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Stock</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="table_produits">
        <property name="gridStyle">
         <enum>Qt::DashDotDotLine</enum>
        </property>
        <column>
         <property name="text">
          <string>Produit</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>En Stock</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Ajouter</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btn_modifier_stock">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white;
border:none;
padding:8px;</string>
          </property>
          <property name="text">
           <string>Modifier</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>A propos</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_adresse">
          <property name="styleSheet">
           <string notr="true">font-size:10px;
font-weight:bold;</string>
          </property>
          <property name="text">
           <string>Point de vente</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_responsable">
          <property name="styleSheet">
           <string notr="true">font-size:10px;</string>
          </property>
          <property name="text">
           <string>Responsable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_contact">
          <property name="styleSheet">
           <string notr="true">font-size:10px;</string>
          </property>
          <property name="text">
           <string>Contact</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Liste des utilisateurs</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="table_users">
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="gridStyle">
           <enum>Qt::DashDotDotLine</enum>
          </property>
          <column>
           <property name="text">
            <string>Nom</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Identifiant</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Mot de passe</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
